javascript/q2.js

```diff
@@ -0,0 +1,34 @@
+function Metadataparser(version, channel, keyField) {
```

Comment:
Metadataparser should be an object.

Reply:
As per question, it should be constructor function